Tools/autotest/arducopter.py

```diff
@@ -267,10 +267,12 @@ def fly_square(self, side=50, timeout=300):
         self.save_wp()
 
         # switch back to stabilize mode
-        self.set_rc(3, 1500)
         self.mavproxy.send('switch 6\n')
         self.wait_mode('STABILIZE')
 
+        # increase throttle because we're about to pitch:
+        self.set_rc(3, 1550)
```

Comment:
@peterbarker Do you remember why you raise throttle ? we are in Loiter, so the copter is just climbing infinitly !